maestrowf/datastructures/core/executiongraph.py

```diff
@@ -602,16 +602,21 @@ def _execute_record(self, record, adapter, restart=False):
     def write_status(self, path):
         """Write the status of the DAG to a CSV file."""
         header = "Step Name,Workspace,State,Run Time,Elapsed Time,Start Time" \
-                 ",Submit Time,End Time,Number Restarts"
+                 ",Submit Time,End Time,Number Restarts, Job ID"
         status = [header]
         keys = set(self.values.keys()) - set(["_source"])
         for key in keys:
             value = self.values[key]
+
+            jobid_str = "Job ID not available"
+            if value.jobid:
+                jobid_str = " ".join(value.jobid)
```

Comment:
A couple things here:
1. `Job ID not available` is a little long for a column. Identifiers are under 10 digits long. That's way over 10. It's probably better to leave the column blank, or have something like `--` (I think the steps that haven't run have something like `--:--:--` for their timestamps). You don't need to be overly verbose, a visual cue is enough to tell a user isn't not available.
2. We should only report the last one. If something is running and has run multiple times, the user is likely to not care about the old identifiers. The most recent one is the most important. In that case, simply report `value.jobid[-1]` if the list is non-empty.

Reply:
Sounds good, I will make that change